web/pages/meetings/weekly.md

```diff
@@ -0,0 +1,27 @@
+title: Weekly PlasmaPy Community Meeting
+author: Erik T. Everson
+hidetitle: True
+
+[Jitsi]: https://meet.jit.si/plasmapy
+[minutes and agenda]: https://drive.google.com/drive/folders/0ByPG8nie6fTPV1FQUEkzMTgtRTg?usp=sharing
+[google calendar]: https://calendar.google.com/calendar?cid=bzVsb3ZkcW0zaWxsam00ZTlrMDd2cmw5bWdAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ
```

Comment:
```suggestion
[google calendar]: https://calendar.google.com/calendar/embed?src=o5lovdqm3illjm4e9k07vrl9mg%40group.calendar.google.com
```
This can be accessed without Google sign-in this way. Also, hey, I didn't know you could do this kind of linking in .md, thanks!